integration-core/src/main/java/org/symphonyoss/integration/core/bridge/IntegrationBridgeExceptionHandler.java

```diff
@@ -140,6 +144,29 @@ public void handleUnexpectedException(Exception e) {
   }
 
   /**
+   * Verify if the error returned by the Agent is related to FORBIDDEN exception.
+   *
+   * @param status HTTP status code
+   * @param responseMessage Message returned by the Agent
+   * @return true if the HTTP status is equals to 403 and HTTP response body contains a JSON
+   * object with code field reporting 403 as well.
+   */
+  private boolean shouldUpdateStreams(Status status, String responseMessage) {
```

Comment:
I believe that an affirmative name would fit better in this method. Something like isAForbiddenError().